src/components/clusterWizard/ClusterWizardStep.tsx

```diff
@@ -35,9 +42,9 @@ const NavItem: React.FC<WizardNavItemProps & { isValid?: () => boolean }> = ({
   if (!isDisabled && !isCurrent && !isValid()) {
     validatedLinkName = (
       <>
-        <ExclamationTriangleIcon
+        <ExclamationCircleIcon
```

Comment:
Wireframes have the icon after the content https://marvelapp.com/prototype/fjj6g57/screen/76752444

Reply:
done